charts/redis/values.yaml

```diff
@@ -9,3 +9,8 @@ redis:
     port: 6379
   db_name: 1
   expiry: 3600
+
+sealed_secrets:
+  enabled: false
+
+application_name:
```

Comment:
Can we put this inside the sealed_secrets ? And can we rename it as `sealed_secret_name` ?

Reply:
Done